update-adr-overview (#162)
wmde/wikit: docs/src/doc/decisions-and-ADRs/01_overview.stories.mdx

# Decisions and ADRs

> Crucial decisions are documented in the form of Architecture Decision Records (ADRs). Although ADRs often cover topics exclusive to the dev realm, the clarity in which they are written and structured makes their content accessible to all the members of our cross-functional team. You can find all development and design related decisions in this folder.

Author: ```suggestion
Crucial decisions are documented in the form of Architecture Decision Records (ADRs). Although ADRs often cover topics from an engineering perspective, the clarity in which they are written and structured makes their relevant content accessible to all the members of our cross-functional team. You can find all development and design related decisions in this folder.
```
